example.docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+Version: '3'
+Service:
+  Pufferpanel:
+    Image: pufferpanel/pufferpanel
+    Container_name: Pufferpanel
```

Comment:
Name is not used by compose anymore

Reply:
https://docs.docker.com/compose/compose-file/compose-file-v3/
Simply untrue.

Reply:
It is true, if swarm mode is used. May not always be true, but it is also not wrong in cases.